Practice2/README.md

```diff
@@ -0,0 +1,78 @@
+# HTML/CSS Practice Two
+
+## Requirements
+* Work fine on Chrome browser's latest version
+* Getting the code to work cross-browser latest versions (Chrome, MS Edge)
+* Use Parcel for development
+
+## Target
+* Apply knowledge to responsive practice one design
+* Used media queries for popular screen size
+
+## Technical
+* HTML5
+* CSS3
+* Parcel
+
+## How to run the project
+*Step one*: Clone the code folder from gitlab to your device
+* Choose a path to save that file -> At that path open the command window
+* Run command
+>> git clone [https://github.com/duylong13/Internship](https://github.com/duylong13/Internship).
+*Step two*: How to run the project
+* Turn on Visual Studio Code. Open the folder you just cloned to your computer.
+* Select Terminal on the toolbar or press Ctrl + Shift + `
+* Run command "cd Practice2"
+* Run command "npm install" to install package of practice
+* Run command "npm start"
+* Hold ctrl and click on the successfully created localhost link to view the website
+
+## Folders Structure
+* src/
+  * assets/
+    * fonts/
+    * images/
+  * styles/
+    * base/
+      * base/
+        * base.css
+      * reset/
+        * reset.css
+      * main-base.css
+    * components/
+      * main-com.css
+    * layout/
+      * choose-your-plan/
```

Comment:
This is component, not layout.
No need to put component-name.css into component-name/.

Reply:
Please update for other components: features, header, footer ...